LoopKitUI/ViewModels/TherapySettingsViewModel.swift

```diff
@@ -20,21 +20,24 @@ public class TherapySettingsViewModel: ObservableObject {
 
     private let initialTherapySettings: TherapySettings
     let pumpSupportedIncrements: (() -> PumpSupportedIncrements?)?
-    let syncPumpSchedule: (() -> PumpManager.SyncSchedule?)?
+    let syncBasalRateSchedule: (() -> PumpManager.SyncBasalRateSchedule?)?
+    let validateTempBasal: (() -> PumpManager.ValidateTempBasal?)?
     let sensitivityOverridesEnabled: Bool
     public var prescription: Prescription?
 
     public init(therapySettings: TherapySettings,
                 supportedInsulinModelSettings: SupportedInsulinModelSettings = SupportedInsulinModelSettings(fiaspModelEnabled: true, walshModelEnabled: true),
                 pumpSupportedIncrements: (() -> PumpSupportedIncrements?)? = nil,
-                syncPumpSchedule: (() -> PumpManager.SyncSchedule?)? = nil,
+                syncBasalRateSchedule: (() -> PumpManager.SyncBasalRateSchedule?)? = nil,
```

Comment:
The types of these args appear to be closures that return closures? Why the nesting?

Reply:
I thought of this.  The ones that themselves return closures are strange, it is true, but remember that's the way we ensure clients of `TherapySettingsViewModel` get updated values.  In practice it is probably unlikely they will need an "updated version of the `SyncBasalRateSchedule` function, but following the pattern is probably a better thing to do.

Reply:
If the extra complexity is not needed, it should probably be simplified. Seeing the double abstraction implies to the reader that it is necessary.

Reply:
Making such a change is unrelated to this PR, though, I think.